CRM/Core/Form/Renderer.php

```diff
@@ -370,7 +370,10 @@ public function addOptionsEditLink(&$el, $field) {
       $path = $field->getAttribute('data-option-edit-path');
       // NOTE: If we ever needed to support arguments in this link other than reset=1 we could split $path here if it contains a ?
       $url = CRM_Utils_System::url($path, 'reset=1');
-      $el['html'] .= ' <a href="' . $url . '" class="crm-option-edit-link medium-popup crm-hover-button" target="_blank" title="' . ts('Edit Options') . '" data-option-edit-path="' . $path . '"><i class="crm-i fa-wrench"></i></a>';
+      $icon = CRM_Core_Page::crmIcon('fa-wrench', ts('Edit %1 Options', [1 => $field->getLabel() ?? ts('Field')]));
```

Comment:
I'd use the Elvis here as the function itself should always be defined we don't need to guard against undefined with ?? and we do want to provide a default in case label is empty (coalesce treats an empty string as having a value, Elvis does not).

```suggestion
      $icon = CRM_Core_Page::crmIcon('fa-wrench', ts('Edit %1 Options', [1 => $field->getLabel() ?: ts('Field')]));
```